### PR TITLE
feat(#minor); Balancer Forks; Blacklist Aave boosted USD Pools

### PR DESCRIPTION
**Pull request by harsh9200**
Fixes: #983

### CODE REVIEW

File: subgraphs/balancer-forks/src/modules/Metrics.ts [hunk 1]
```diff
@@ -24,7 +18,7 @@ import * as constants from "../common/constants";
 import { updateRevenueSnapshots } from "./Revenue";
 
 export function updateUsageMetrics(block: ethereum.Block, from: Address): void {
-  const account = getOrCreateAccount(from.toHexString());
+  getOrCreateAccount(from.toHexString());
```
[the remark this answers is not in the document]
Comment (harsh9200) [reply subgraphs/balancer-forks/src/modules/Metrics.ts:21]: yeah, we just need to initialize the `Account` entity.

File: subgraphs/balancer-forks/src/modules/Revenue.ts [hunk 2]
```diff
@@ -27,76 +28,69 @@ export function updateRevenueSnapshots(
 
   const totalRevenueUSD = supplySideRevenueUSD.plus(protocolSideRevenueUSD);
 
+  // Protocol metrics
+  if (
```
[the remark this answers is not in the document]
Comment (harsh9200) [reply subgraphs/balancer-forks/src/modules/Revenue.ts:32]: The blacklisting is done in such a way that the revenue of the blacklisted protocol is not included in the overall protocol metrics. The pool is not blacklisted completely.

File: subgraphs/balancer-forks/src/prices/config/mainnet.ts [hunk 3]
```diff
@@ -68,6 +68,9 @@ export const HARDCODED_STABLES: Address[] = [
   Address.fromString("0x652d486b80c461c397b0d95612a404da936f3db3"), // bb-a-USDC
   Address.fromString("0xa3823e50f20982656557a4a6a9c06ba5467ae908"), // bb-a-DAI
   Address.fromString("0xe6bcc79f328eec93d4ec8f7ed35534d9ab549faa"), // bb-a-USDT
+  Address.fromString("0x2f4eb100552ef93840d5adc30560e5513dfffacb"), // bb-a-USDT
```
[the remark this answers is not in the document]
Comment (harsh9200) [reply subgraphs/balancer-forks/src/prices/config/mainnet.ts:71]: yeah I will fix that

File: subgraphs/balancer-forks/src/modules/Metrics.ts [hunk 4]
```diff
@@ -203,13 +197,13 @@ export function updateSnapshotsVolume(
   volumeUSD: BigDecimal,
   block: ethereum.Block
 ): void {
-  let protcol = getOrCreateDexAmmProtocol();
```
[the remark this answers is not in the document]
Comment (harsh9200) [reply subgraphs/balancer-forks/src/modules/Metrics.ts:206]: Nice catch!!

**Review by steegecs — APPROVED** [review]
LGTM

**Pull request merged**